src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -524,7 +523,8 @@ def estimateTime(self, remaining=None):
 
         stitch_time = (self._nx * self._ny * max_pxs * self._overlap) / self.STITCH_SPEED
         try:
-            move_time = max(self._guessSmallestFov(self._streams)) * (remaining - 1) / self._move_speed
+            # Extra time to account for actual stage movement happening in more than 2 dimensions
+            move_time = max(self._guessSmallestFov(self._streams)) * (remaining - 1) / self._move_speed  + 0.3 * remaining
```

Comment:
Maybe use estimateMoveDuration() here? 
Also, I'm not certain that "remaining - 1" is correct, but I would think it should be the same number as for the "0.3 -...". So either use on both sides "remaining - 1" or "remaining" or something more clever. 
Maybe the "- 1" was to handle the case when a 1x1 overview is acquired?! But that would not be really explaining it for the most standard cases....